pastas/model.py

```diff
@@ -1475,17 +1476,18 @@ def _get_file_info(self):
 
         return file_info
 
-    def fit_report(self, output="basic", warnbounds=True):
```

Comment:
I think we should add a DeprecationError for warnbounds here, for the off-chance people use it.

e.g., "the 'warnbounds' argument is deprecated, please use 'warnings=True' now.

Reply:
added